.travis.yml

```diff
@@ -17,7 +17,7 @@ matrix:
     osx_image: xcode9.4  # 10.13
     language: generic
     script:
-    - "./.travis/tests.sh"
+    - coverage run "./.travis/tests.sh"
```

Comment:
I think the `coverage run` bit should be inside `.travis/tests.sh`, probably replacing this line:

`python3 ./tests/runAllTests.py`

by

`coverage run ./tests/runAllTests.py`